### ISSUE TITLE
[Feature Request] Signos de puntuación iniciales for spanish users.

### ISSUE THREAD

**Issue by MrMxffin** [TodePond/DreamBerd#579]
Spanish users (and others) should be able to use the inverted question mark "¿" and inverted exclamation mark "¡". If inverted punctuation is used, it must match the counterparts at the end.

**Comment by Alevale** [comment]
This should be like the currency for interpolation and it should give a compilation error if not used and it's a rule of the language of the location you are in.

```dreamberd
print("Hello World")! // This is fine in most countries

¡print("Hello World")! // This is mandatory in South America and Spain. (otherwise compilation fails)
```

**Comment by Alevale** [comment]
> This might clash with negative priority, which already uses the inverted exclamation mark.

WOWOW, is that actually a thing? (sorry I didn't see it in the readme 🤔 ) do you happen to have a link?

**Comment by mybearworld** [comment]
> do you happen to have a link?

Sure, last code block here: https://github.com/TodePond/DreamBerd?tab=readme-ov-file#overloading

https://github.com/TodePond/DreamBerd/blob/4700ff9c358cb340083077104855e8c9eed9c4f9/README.md?plain=1#L541-L547

**Comment by ktlknss** [comment]
No, putting an inverted exclamation mark at the start of a line inverts the inversion adding priority
